package mvp.pipeline;

message OrbitalImage {
  required string camera_path = 1;
  required string image_path = 2;
}
